draft-aegis-mimi-arch-latest.md

```diff
@@ -348,11 +347,13 @@ ApplicationID is defined as a UUIDv4 string and MUST be unique across all
 gateways that are federated with one another. // TODO: Link to UUIDv4 definition
 
 ~~~
-struct {
-    UUIDv4 id;
-    IdentityConfiguration identity_configuration;
+ApplicationConfiguration
+{
+    "identity_configuration": IdentityConfiguration,
+    "id": UUIDv4
+    // TODO: Could apps use multiple IdentityConfigurations?
```

Comment:
I think they are composable so maybe you would just make MyAppIdentity that just has multiple identity systems internally.